eslint.config.ts

```diff
@@ -99,10 +99,6 @@ export default tseslint.config(
 		files: ["**/*.{yml,yaml}"],
 		rules: {
 			"yml/file-extension": ["error", { extension: "yml" }],
-			"yml/sort-keys": [
-				"error",
-				{ order: { type: "asc" }, pathPattern: "^.*$" },
-			],
```

Comment:
😄 I've been thinking of getting rid of this in CTA. I like it but recognize it's pretty low impact & takes up space...

Reply:
Heh, I don't like having to use any orderings other than alphabetical/lexicographic. There's an unbounded set of reasonable ways to order things like yaml. I personally can never keep track of them in my head. Putting name, then triggers, then the rest is one semi-arbitrary preference that I used to have myself, then would often slip up on.